src/dispatch/static/dispatch/src/signal/SignalInstanceTab.vue

```diff
@@ -9,12 +9,6 @@
     <template v-slot:item.signal="{ item }">
       <signal-popover v-model="item.signal" />
     </template>
-    <template v-slot:item.duplication_rule="{ item }">
-      <v-simple-checkbox :value="item.duplication_rule || false" disabled></v-simple-checkbox>
-    </template>
-    <template v-slot:item.suppression_rule="{ item }">
-      <v-simple-checkbox :value="item.supression_rule || false" disabled></v-simple-checkbox>
-    </template>
     <template v-slot:item.created_at="{ item }">
```

Comment:
```suggestion
    <template v-slot:item.entities="{ item }">
      <v-row>
        <v-chip v-for="(entity, index) in item.entities" :key="index" class="mr-2">
          {{ entity.entity_type.name }}
        </v-chip>
      </v-row>
    </template>
    <template v-slot:item.created_at="{ item }">
```

Maybe something like this, shows `object Object` right now

Reply:
Something like this is probably better if we have a lot of them:

```vue
<template v-slot:item.entities="{ item }">
  <template v-for="(entity, index) in item.entities.slice(0, 3)">
    <v-chip :key="index" label>{{ entity.entity_type.name }}</v-chip>
  </template>
  <v-chip v-if="displayEntityCount(item.entities)" label>+{{ item.entities.length - 3 }}</v-chip>
</template>

<script>
export default {
  // ...
  methods: {
    displayEntityCount(entities) {
      return entities.length > 3;
    },
  },
};
</script>
```